src/main/java/uk/gov/digital/ho/pttg/ArchiveService.java

```diff
@@ -31,8 +32,13 @@ public ArchiveService(AuditEntryJpaRepository repository, ObjectMapper objectMap
         this.objectMapper = objectMapper;
     }
 
-    public void archiveResult(LocalDate resultDate, String result, List<String> eventIds, LocalDate lastArchiveDate) {
-
+    public void handleArchiveRequest(LocalDate resultDate, String result, List<String> correlationIds, LocalDate lastArchiveDate, String nino) {
+        LocalDateTime lastArchiveTime = lastArchiveDate.atTime(23, 59, 59, 999999999);
+        if (repository.countNinosAfterDate(lastArchiveTime, nino) > 0) {
+            return;
```

Comment:
I think logging that the audit request is being ignored because of the newer entry would be helpful.